packages/sanity/src/_internal/cli/actions/manifest/extractManifestAction.ts

```diff
@@ -90,7 +91,7 @@ async function extractManifest(
     const workspaceFiles = await writeWorkspaceFiles(workspaceManifests, staticPath)
 
     const manifest: CreateManifest = {
-      version: 1,
+      version: 2,
```

Comment:
Lets add a commnet: `//Increment this if whenever we change the API`

Q: should we also add a comment about version history? Ie:
```
/* 
Version history:
1: Initial release.
2: Added tools file.
*/
```